core/templates/dev/head/pages/exploration_player/feedback_popup_directive.html

```diff
@@ -35,7 +35,7 @@
     </md-button>
     <div class="checkbox" ng-show="isLoggedIn">
       <label>
-        <input type="checkbox" ng-model="isSubmitterAnonymized">
+        <input type="oppia-stay-anonymous-checkbox" ng-model="isSubmitterAnonymized">
```

Comment:
@jeeb0xical, you are changing the `type` attribute of input, try changing the class name of div in line 36.